lib/sql.js

```diff
@@ -1177,6 +1260,24 @@ SQLConnector.prototype.buildFields = function(model, data, excludeIds) {
   return this._buildFieldsForKeys(model, data, keys, excludeIds);
 };
 
+/**
+ * Build an array of fields for the database operation from data array
+ * @param {String} model Model name
+ * @param {Object} data Array of Model data object
+ * @param {Boolean} excludeIds Exclude id properties or not, default to false
+ * @returns {[{names: Array, values: Array, properties: Array}]}
+ */
+SQLConnector.prototype.buildFieldsFromArray = function(model, data, excludeIds) {
+  const fields = [];
+  if (data.length > 0) {
```

Comment:
I would prefer to use `Array.isArray(data) && data.length > 0`. For example, if data is string, data.length exists too.

Reply:
Done. 👍